packages/tofu/src/util.js

```diff
@@ -58,34 +85,62 @@ function getNameFromNode(node) {
   }
 }
 
+/**
+ *
+ * @param {import("@babel/types").Node} node
+ * @returns {node is import("./inspectPrimordialAssignments").MemberLikeExpression}
+ */
 function isMemberLikeExpression(node) {
   return (
     node.type === 'MemberExpression' || node.type === 'OptionalMemberExpression'
   )
 }
 
+/**
+ *
+ * @param {import("./inspectPrimordialAssignments").MemberLikeExpression} node
+ * @returns {node is import("./inspectPrimordialAssignments").NonComputedMemberLikeExpression}
+ */
 function isNonComputedMemberLikeExpression(node) {
   return !node.computed && isMemberLikeExpression(node)
 }
 
+/**
+ *
+ * @param {import("@babel/types").LVal} identifierNode
```

Comment:
Note: I guess I'd expect `prettier` to make the quotes consistent in these types, but maybe it doesn't do that out-of-the-box.  [prettier-plugin-jsdoc](https://npm.im/prettier-plugin-jsdoc) seems like it could be helpful.